conf/services/changesetReplacementOps.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "ChangesetReplacement",
+  "label": "ChangesetReplacement",
+  "members": [
+    {
+      "description": "No feature outside or overlapping the bounds will be modified. Reference line features may be split at the specified bounds.",
+      "default": "false",
+      "input": "checkbox",
+      "type": "bool",
+      "id": "strictBounds",
+      "label": "Use strict bounds handling"
+    },
+    {
+      "description": "All reference features within the specified bounds will be removed.",
+      "default": "false",
+      "input": "checkbox",
+      "type": "bool",
+      "id": "fullReplacement",
+      "label": "Full replacement"
+    },
+    {
+      "description": "Disables the snapping of unconnected ways before changeset derivation.",
+      "default": "false",
+      "input": "checkbox",
+      "type": "bool",
+      "id": "disableWaySnapping",
+      "label": "Disable way snapping"
+    },
+    {
+      "description": "Disables the conflation of data cut out during replacement and the replacement data.",
+      "default": "false",
+      "input": "checkbox",
+      "type": "bool",
+      "id": "disableConflation",
+      "label": "Disable conflation"
+    }
+  ]
```

Comment:
The UI form metadata for derive changeset replacement advanced options.  Some use a script to generate this from ConfigOptions.asciidoc, but these parameters are just flags and do not carry a value so they are not documented in that file, hence it's hardcoded.